src/Cache/Cache.php

```diff
@@ -168,7 +168,21 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
+                $registry->set($name, new NullEngine());
+                trigger_error($e->getMessage(), E_USER_WARNING);
+
+                return;
+            }
+            $fallbackEngine = clone static::engine($config['fallback']);
+            $copyableConfig = ['groups' => null, 'prefix' => null];
+            $fallbackEngine->init(array_intersect_key($config, $copyableConfig));
```

Comment:
There are a couple currently:

- `testCacheFallbackWithGroups`: tests original config is still associated with its groups, fallback config is associated with only its original groups
- `testCacheFallbackIntegration`: tests clearing group of original config causes reading from that config's fallback fails properly (when reading from original config)

I will add some more that write/clear groups both in the original and the fallback to make sure they don't get confused somehow.